Acrolinx quality improvements: cprofile.md (#6936)
MicrosoftDocs/windowsserverdocs: WindowsServerDocs/administration/windows-commands/cprofile.md

---
title: cprofile

> description: Reference article for the cprofile command. This command is deprecated and is not guaranteed to be supported in future releases of Windows.

Reviewer: ```suggestion
description: Reference article for the cprofile command. The command cprofile is deprecated and maybe removed them from a future releases of Windows.
```

ms.topic: reference
ms.assetid: d83742c0-9c38-4d24-8f63-2eaad0c3c382
ms.author: jgerend
author: JasonGerend
manager: mtillman
ms.date: 01/03/2023
---

# cprofile

Applies to: Applies to: Windows Server (All supported versions)

> The command cprofile is deprecated and is not guaranteed to be supported in future releases of Windows.

Reviewer: ```suggestion
The command cprofile is deprecated and maybe removed them from a future releases of Windows.
```
